src/SmoFacade/Listener.cs

```diff
@@ -89,6 +98,8 @@ public Listener(SqlConnectionStringBuilder connectionStringBuilder, string crede
 
     public AvailabilityGroup AvailabilityGroup { get; set; }
 
+    public IEnumerable<Server> AllInstances => _allInstances;
+

```

Comment:
You might be able to do a join b/w primaries and secondaries - not 100% sure tho??

Reply:
I removed this since it is no longer needed.

Reply:
Yeah I'm not sure we need an additional variable for this.  You should be able to join the primary and secondary members inline as a property